packages/resource-deployment/scripts/run-command-when-batch-nodes-are-idle.sh

```diff
@@ -117,7 +128,11 @@ function runCommand() {
     echo "Logging into '$batchAccountName' Azure Batch account"
     az batch account login --name "$batchAccountName" --resource-group "$resourceGroupName"
 
-    pools=$(az batch pool list --query "[].id" -o tsv)
+    if [[ -z "$poolId" ]]; then
+        pools=$(az batch pool list --query "[].id" -o tsv)
```

Comment:
does it return an array? if yes we need normalize parameter to accept array as well